platform/entanglement/src/services/LinkService.js

```diff
@@ -39,42 +39,28 @@ define(
         }
 
         LinkService.prototype.validate = function (object, parentCandidate) {
-            if (!parentCandidate || !parentCandidate.getId) {
-                return false;
-            }
-            if (parentCandidate.getId() === object.getId()) {
-                return false;
-            }
-            if (parentCandidate.getModel().composition.indexOf(object.getId()) !== -1) {
-                return false;
-            }
-            return this.policyService.allow(
-                "composition",
-                parentCandidate.getCapability('type'),
-                object.getCapability('type')
-            );
+            var objectId = object.getId();
```

Comment:
Thanks-- I appreciate not changing code that is not related to the intended changes if it's not mandated by the style guide.

There is no semantic difference between the two methods-- they validate whether the operation can be performed on the two objects.  That said, link service and move service should have different criteria for validation.  At some level, documentation of that behavior (e.g. can't move immutable/non-creatable objects) is useful.  

I realize our current implementation may not have significant differences, this JSDoc may be written when #129 is implemented.